houston/ardu/connection.py

```diff
@@ -1,21 +1,24 @@
 __all__ = ['MAVLinkMessage', 'CommandLong', 'MAVLinkConnection']
 
+from typing import Any, List, Callable, Dict
 import pymavlink
+from pymavlink.mavutil import mavlink
 import attr
+import dronekit
 
 from ..connection import Message, Connection
 
 
-class MAVLinkMessage(Message):
+class MAVLinkGeneralMessage(Message):
```

Comment:
Just curious: why the change of name?

Reply:
Because the messages we send and receive are different but we want them to have a common parent.